ycast/my_stations.py

```diff
@@ -71,17 +69,6 @@ def get_stations_by_category(category):
     if my_stations_yaml and category in my_stations_yaml:
         for station_name in my_stations_yaml[category]:
             station_url = my_stations_yaml[category][station_name]
-            station_id = str(get_checksum(station_name + station_url)).upper()
+            station_id = str(generic.get_checksum(station_name + station_url)).upper()
```

Comment:
Why move get_checksum to generic?
This should be specific to stations.

Reply:
Because it is already used in my_stations

Reply:
Ah, my bad. You're right.